### ISSUE TITLE
DeepSpeech model accuracy

### ISSUE THREAD

**Issue by BuaaAlban** [tensorflow/models#7567]
The speech trained by the code released here cant' reach high accuracy, WER reach to about 0.20,
and I found  a problem that it can't translate words like "moss-->mos  mirror-->miror", any repeated character can't be translated right.

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
What is the top-level directory of the model you are using
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by BuaaAlban** [comment]
> Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
> What is the top-level directory of the model you are using
tensorflow/models/deep_speech

> Have I written custom code
Yes

> OS Platform and Distribution
Ubuntu 16
> TensorFlow installed from
pip 
> TensorFlow version
1.14
> Bazel version
???
> CUDA/cuDNN version
10.0 /7.5.0
> GPU model and memory

> Exact command to reproduce

**Comment by YuanEric88** [comment]
@BuaaAlban 

Hi,  I am also training deepspeech by libirspeech dataset, and I have some questions as follow:
1, What is the training dataset in your experiment? Is it the combination of train-clean-100, train-clean-360 and train-other-500? or just part of them?
2, In which dataset did you achieve WER=0.2, dev-clean only, dev-clean + dev-other, or test-clean?
3, what is the training strategy of your experiment? Specifically, the value of learning_rate, training_epochs, learning_rate_decay strategy, etc.

Looking forward to your reply. Thanks.

**Comment by BuaaAlban** [comment]
> @BuaaAlban
> 
> Hi, I am also training deepspeech by libirspeech dataset, and I have some questions as follow:
> 1, What is the training dataset in your experiment? Is it the combination of train-clean-100, train-clean-360 and train-other-500? or just part of them?
> 2, In which dataset did you achieve WER=0.2, dev-clean only, dev-clean + dev-other, or test-clean?
> 3, what is the training strategy of your experiment? Specifically, the value of learning_rate, training_epochs, learning_rate_decay strategy, etc.
> 
> Looking forward to your reply. Thanks.

I have used the default config.  The training set should be the combination of the training set and test it on dev clean or test clean, sorry that I have forgotten the details. I have tried different optimizers like Adam and SGD
